Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
```

Comment:
That's because Rails will automatically `require` things in the Gemfile, but not things from the gemspec. If you `require 'pry'; binding.pry` that should work.